### PR TITLE
Adjust the affects of adrenaline to better fit how it acts in real life

### PR DESCRIPTION
**Pull request by HotCupOfJava**
#### Summary

SUMMARY: Balance "Adjust the affects of adrenaline to reflect real life rather than magic"

#### Purpose of change

The current implementation of adrenaline makes it a magical "never get winded" type drug that doesn't reflect the actual affects of most commonly found in medical experiments.

#### Describe the solution

It adjusts the primary effects from the hardcoded adrenaline effect to a flat stim boost that could reach dangerous levels if taken with other stimulants. It also acts as a painkiller.

#### Describe alternatives you've considered

Add implementation of allergens and allow it to treat anaphylactic shock

#### Testing

I spawned in the autoinjector and injected it in myself, the effects kicked in afterwords, I injected myself with it again and overdosed as accurate to real life.

#### Additional context

This should temporarily rebalance it until a proper stim balancing and adrenaline system can be implemented.

### CODE REVIEW

**Comment by Gourmand-is-OP** [comment]
This is less accurate than the original adrenaline effect. Adrenaline is responsible for increasing heart rate, blood pressure, air intake, and forcing blood to the muscles and brain. I.E., stamina regeneration. These are similar, but significantly different to stimulants like caffeine, which primarily effects the brain, and not the rest of the body. However, the part about anaphylactic shock is correct, and if implemented, would be a good inclusion.

This also fails to balance adrenaline. It is nerfed into the ground, making it next to useless, given that the benefits of stimulants are significantly lower in large amounts. This is not reasonable, given the present drawback, rarity of the item, and permanent cost of the trait. It perhaps should decrease the overdose threshold, but counting as normal stimulants itself makes the effect far weaker.

Most importantly, this is less fun. The original effect of adrenaline is simply more interesting to play with. This is a subjective opinion, but having a common stimulant effect in place of a specialized one with both strong positives and drawbacks, is much less engaging.

Finally, adult adrenaline auto-injectors come int 0.3mg sizes, and standard doctor procedure looks to be 0.5mg (for those over 120lbs) per 5 minutes as needed. This would imply that a significantly greater number of injections would be required to overdose.

**Comment by kevingranade** [comment]
Accelerated stamina regeneration (along with some stimulant effect) sounds better to me too, but as I don't believe we have that as an effect right now then a moderate stimulant effect is the closest we're able to come up with.
It doesn't make sense to "rebalance" adrenaline as being particularly potent, the current state of adrenaline is not de facto correct in any meaningful way and the rarity of an item is irrelevant to it's efficacy.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -94,16 +94,21 @@
     "id": "adrenaline_injector",
     "type": "COMESTIBLE",
     "comestible_type": "MED",
-    "name": { "str": "syringe of adrenaline", "str_pl": "syringes of adrenaline" },
-    "description": "A syringe filled with a shot of adrenaline.  It serves as a powerful stimulant when you inject yourself with it.  Asthmatics can use it in an emergency to clear their asthma.",
+    "name": { "str": "epinephrine autoinjector", "str_pl": "epinephrine autoinjectors" },
```
Comment (kevingranade) [inline data/json/items/comestibles/med.json:97]:
```suggestion
    "name": { "str": "epinephrine autoinjector" },
```

**Comment by Gourmand-is-OP** [comment]
We could probably reuse "stamina_regen_modifier" from the fast metabolism mutation to increase stamina regen. I also wouldn't in a million years describe adrenaline as "moderate". 
The current effect also already gives accelerated stamina regeration, and the point of this push is to remove the stamina regeneration effect, with it being unrealistic. (Although I assert that that effect is one of the most realistic things about it.)
As it is, the current, short term buffs to physical stats, speed, and stamina regeneration, are an almost spot on representation of its effect in real life, and this change would remove most of these effects, just in order to make adrenaline less compatible with other stimulants. Such an effect could easily be obtained by altering the injetor and trait to add a stimulant effect along with the adrenal one.

**Pull request merged**